jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshTemplate.java

```diff
@@ -40,14 +41,18 @@
         static final String CMD_DELIMITER = "; ";
     }
 
-    public SshTemplate(SshSetting sshSetting, String timestampFormat, RollingLogResolver rollingLogResolver) throws IOException {
+    public SshTemplate(SshSetting sshSetting, String timestampFormat, RollingLogResolver rollingLogResolver) {
         isTrue(!isEmpty(sshSetting.getLogPath()), "log path must not be null or empty");
         isTrue(!isEmpty(timestampFormat), "timestamp format must not be null or empty");
 
         this.sshSetting = sshSetting;
         this.timestampFormat = timestampFormat;
         this.rollingLogResolver = rollingLogResolver;
-        this.getLogBetweenTimestampsCommand = copyToString(new ClassPathResource("get-log-between-timestamps-template.awk").getInputStream(), defaultCharset());
+        try {
```

Comment:
We could probably get around this with `@SneakyThrows(IOException.class)` - but might get a worse resulting exception type than `UncheckedIOException` as a result - IMO worth the payoff in saving try/catch block, what do others think?

Reply:
I vote for specific wrapper before generic.

Reply:
I am merging this as it is since it seems no one seems to care strongly about this. Thanks for the patience and the PR 🙂